application/config/config.go

```diff
@@ -866,6 +884,28 @@ func (c *Config) IsFedramp() bool {
 	host := strings.ToLower(u.Hostname())
 
 	// fedramp instance should have the format https://*.snykgov.io
-	snykgovDomain := strings.HasSuffix(host, ".snykgov.io")
-	return snykgovDomain
+	return isFedrampHost(host)
+}
+
+func (c *Config) GetSnykCodeApi() (string, error) {
+	if !c.IsFedramp() {
+		return c.SnykCodeApi(), nil
+	}
+
+	// api url is fedramp, but code api url hasn't been updated properly
+	if strings.Contains(c.SnykCodeApi(), "deeproxy.") {
+		snykCodeApiUrl, err := getCodeApiUrlFromCustomEndpoint(c.SnykApi(), c.IsFedramp())
+		if err != nil {
+			return "", err
+		}
+		c.SetSnykCodeApi(snykCodeApiUrl)
+	}
+
+	if c.Organization() == "" {
+		return "", errors.New("Organization must be present in a fedramp environment")
+	}
+
+	path := "/org/" + c.Organization() + "/code"
```

Comment:
Will that work for when both org slug and org id are provided?

Reply:
`c.Organization()` will always return the Org ID, no matter what the user specified. So this is fine!

Reply:
@PeterSchafer Thanks!

Can we document that in Code please? 🙏